### ISSUE TITLE
Object doesn't support property or method 'values'

### ISSUE THREAD

**Issue by maskaravivek** [awslabs/amazon-quicksight-embedding-sdk#25]
I am using the SDK on IE 11 and am getting the following error. 

```
TypeError: Object doesn't support property or method 'values'
  at event(./node_modules/amazon-quicksight-embedding-sdk/dist/lib/constructEvent.js:13:1)
  at sendInitialPostMessage(./node_modules/amazon-quicksight-embedding-sdk/dist/EmbeddableDashboard.js:213:1)
```

Can someone help?

**Comment by chillozz** [comment]
@maskaravivek Hello!

**Comment by maskaravivek** [comment]
It would be great if it can be merged and the library version be bumped. :)

This is the only blocker for us to use this SDK on production.

**Comment by chillozz** [comment]
It is great idea, because i don't have access to merge request. I am waiting response from Amazon support.

**Comment by cxiangna** [comment]
This issue has been fixed in verison 1.0.5, resolving.
